File: Data Structures & Algorithms/counting-bits/submission-0.cpp
class Solution {
public:
    vector<int> countBits(int n) {
        // 1 1 2 1 2 2 3 1 2 2 3 
        vector<int> otpt(n+1,0);
        for(uint i = 0;i <= n; i++) {
            otpt[i] = popcount(i);
        }
        return otpt;
    }
};

File: Data Structures & Algorithms/anagram-groups/submission-0.py
class Solution:
    def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
        mp = {}
        for i in strs:
            l = [0 for i in range(26)]
            for j in i:
                l[ord(j)-ord('a')]+=1;

            k  =tuple(l)
            if(mp.get(k,-1) == -1):
                mp[k] = [i]
            else:
                mp[k].append(i)
        otpt = []
        for i in mp:
            otpt.append(mp[i])
        return otpt